__tests__/integration/one-app.spec.js

```diff
@@ -178,7 +178,7 @@ describe('Tests that require Docker setup', () => {
         },
       });
 
-      expect(response.status).toBe(200);
+      expect(response.status).toBe(404);
```

Comment:
This was only 200 because the application error caused by this bug was not setting the HTTP status and it defaulted to 200. Fixed in https://github.com/americanexpress/one-app/pull/1243

Reply:
For some more background this test did not change when we migrated from express to fastify. However, it was correct in express. The `cors` express middleware does not change the HTTP status on preflight rejection and it defaults to 200. The `@fastify/cors` package calls `reply.callNotFound` on preflight rejection.